src/Keboola/StorageApi/Client.php

```diff
@@ -491,6 +491,8 @@ public function dropBucket($bucketId, $options = [])
 
         $filteredOptions = array_intersect_key($options, array_flip($allowedOptions));
 
+        // async=true is enforced
+        $filteredOptions['async'] = true;
```

Comment:
ked to tu vynucujes tak uz to nemusis pridavat inde nie ?

Reply:
no vlastne jo. nejdriv jsem to pridal, pak jsem to enforcnul. jdu to vsude zrusit